deploy/helm/node-collector-config.yaml

```diff
@@ -422,7 +460,21 @@ service:
         - k8sattributes
         - batch
       receivers:
-        - receiver_creator
+        - receiver_creator/discovery
+{{- end }}
+{{- if .Values.otel.metrics.enabled }}
+    metrics/node:
+      exporters:
+        - otlp
+      processors:
+        - memory_limiter
+        # TODO - copy all the processors that are needed to process cAdvisor and kubelet metrics
```

Comment:
This is task to do. We need to copy big part of the processors and the pipeline from metrics collector